src/devices/ovrheadset/CMakeLists.txt

```diff
@@ -13,6 +13,12 @@ yarp_prepare_plugin(ovrheadset
 if(NOT SKIP_ovrheadset)
   set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
+  if(YARP_EXPERIMENTAL_API)
+    option(OVR_DEVICE_EXERIMENTAL "experimental version of OvrDevice" OFF)
```

Comment:
I'd rather move this option definition together with the `YARP_EXPERIMENTAL_API` option.
For coherence, and for grouping with cmake-gui, this option should start with `YARP_`.
Also you should be using `cmake_dependent_option`, so that the option is automatically hidden and set to false when you disable `YARP_EXPERIMENTAL_API`

Reply:
(Also the value of this option should be printed in the output of `cmake`)